From 18154a61326c55b7f95c60e6a2e3138d680ef647 Mon Sep 17 00:00:00 2001
From: Madhu <enometh@net.meer>
Date: Tue, 24 May 2022 01:24:23 +0530
Subject: [PATCH 2/5] xpra/sound/sink.py: (SoundSink.__init__): start off with
 a non-zero volume

application doesn't show up on pulse playback
---
 xpra/sound/sink.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/xpra/sound/sink.py b/xpra/sound/sink.py
index 8876b09..afd254e 100755
--- a/xpra/sound/sink.py
+++ b/xpra/sound/sink.py
@@ -129,7 +129,7 @@ class SoundSink(SoundPipeline):
                                           "max-size-bytes=0",
                                           "max-size-time=%s" % QUEUE_TIME,
                                           "leaky=%s" % QUEUE_LEAK]))
-        pipeline_els.append("volume name=volume volume=0")
+        pipeline_els.append("volume name=volume volume=0.1")
         if CLOCK_SYNC:
             if not has_plugins("clocksync"):
                 log.warn("Warning: cannot enable clocksync, element not found")
-- 
2.35.1.dirty

